lib/serverFactory.js

```diff
@@ -6,11 +6,13 @@
 
```

Comment:
I would rather call this httpServerFactory - it is implied http can support both http and https.

I can approve once that is done.